.github/workflows/snapshot-firestore.yml

```diff
@@ -0,0 +1,21 @@
+name: Snapshot firestore
+on:
+  schedule:
+    # At 19:00 UTC / 7:00 PST on every day-of-week from Monday through Thursday.
+    - cron: '0 19 * * 1-4'
+
+jobs:
+  ci:
+    name: Run gcloud to snapshot
+    runs-on: ubuntu-latest
+    steps:
+      - name: Snapshot
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
```

Comment:
Would you mind updating this and the continuous-integration workflow to use the 0.1.1 tag
https://github.com/GoogleCloudPlatform/github-actions/releases

I think that's what we're supposed to do after looking at the OSPO policy today.

Reply:
Done